evap/grades/templates/grades_course_base.html

```diff
@@ -1,11 +1,12 @@
 {% extends "grades_semester_base.html" %}
 
 {% load i18n %}
+{% load morefilters %}
 
 {% block subtitle %}
     {{ block.super }}
     {% if course %}
-    	{% if disable_breadcrumb_course %}
+    	{% if disable_breadcrumb_course or is_grade_publisher|is_false %}
```

Comment:
i noticed that disable_breadcrumb_course is set in exactly one place, and there it is always set to true. please check that and check whether it's necessary to do this check here.